lib/maily.rb

```diff
@@ -29,7 +31,7 @@ def load_emails_and_hooks
       end
 
       # Load hooks
-      hooks_file_path = File.join(Rails.root, hooks_path)
+      hooks_file_path = File.join(Rails.root, hooks_path || DEFAULT_HOOKS_PATH)
```

Comment:
hey @AlexLarra let's try with Maily.init! just before iterating over the mailers (inside the generator). Thanks!